### ISSUE TITLE
Stale readme (GDAX)

### ISSUE THREAD

**Issue by redharp** [alpacahq/marketstore#307]
As a 3rd party developer there is a reasonable expectation of accuracy when reading the published api documentation and additional tools from same org keep valid URL’s and service references.

Expected:
- this repo and readme linked via Alpaca docs would be valid

Actual:
- GDAX still referenced working and dead links to this service (the coinbase pro rebrand > 1year ago)

**Comment by Darkbladecr** [comment]
You are right the readme is stale, but the code is still updated to use the new api url.
https://github.com/alpacahq/marketstore/blob/master/contrib/gdaxfeeder/gdaxfeeder.go#L59

**Comment by goodboy** [comment]
It doesn't work at the moment though. I'm getting a cert auth error from the container:

```python
Failed to create bgworker: Get "https://api.pro.coinbase.com/products": x509: certificate signed by unknown authority
```
